clang/include/clang/Frontend/CompilerInvocation.h

```diff
@@ -224,19 +233,20 @@ class CompilerInvocation : public CompilerInvocationBase {
   /// @{
   // Note: These need to be pulled in manually. Otherwise, they get hidden by
   // the mutable getters with the same names.
-  using CompilerInvocationBase::getLangOpts;
-  using CompilerInvocationBase::getTargetOpts;
-  using CompilerInvocationBase::getDiagnosticOpts;
-  using CompilerInvocationBase::getHeaderSearchOpts;
-  using CompilerInvocationBase::getPreprocessorOpts;
   using CompilerInvocationBase::getAnalyzerOpts;
-  using CompilerInvocationBase::getMigratorOpts;
   using CompilerInvocationBase::getAPINotesOpts;
   using CompilerInvocationBase::getCodeGenOpts;
+  using CompilerInvocationBase::getDebugOpts;
```

Comment:
Can you undo the reordering here? Let's just add the debug options into the right place and then optionally extract this change into a separate trivially NFC commit (and maybe reorder the member variables to match).

Reply:
That was clang-format. I just happened to change one line in there and it reordered everything. I will undo the change.

Reply:
done